src/index.d.ts

```diff
@@ -0,0 +1,67 @@
+export interface ContractAbiEntry {
```

Comment:
What are these declarations?

Reply:
They are subtypes of the contract interface.

Reply:
They are just some basic typings for the Truffle artifacts being exported.

Reply:
Aha! cool. Would it be worth it to use some other NPM dependency for that. Maybe https://github.com/ethereum-ts/truffle-typings or https://www.npmjs.com/package/truffle-contract-typescript

Reply:
Yes, It would be great to use an NPM dependency for that! However, these seem to be for using contracts with truffle. The type definitions are just for the truffle artifact JSON (the JSON files located at `build/contracts/*.json`) - we need this because we re-export the JSON documents.

This should go away once we start using TypeScript, since if you `tsc --declaration` they get get generated for you, so I am not too worried.